src/components/display/PersonCard/PersonCard.tsx

```diff
@@ -68,12 +68,16 @@ export const PersonCard: React.FC<PersonCardProps> = ({
     <div className={container} onClick={() => onClick && onClick()}>
       <Stack direction="row" align="center" justify="space-between">
         <Stack direction="row" gap={16}>
-          <Stack direction="row" gap={12}>
-            {onDelete ? <Button startIcon={Trash} onClick={onDelete} /> : null}
-            {onEdit ? (
-              <Button startIcon={PencilSimple} onClick={onEdit} />
-            ) : null}
-          </Stack>
+          {onDelete || onEdit ? (
```

Comment:
To remove an empty div and unnecessary grid spacing when the delete and edit buttons are absent